docs/.vuepress/config.js

```diff
@@ -205,6 +205,7 @@ module.exports = {
             '/3.0.0-beta.x/guides/custom-admin',
             '/3.0.0-beta.x/guides/client',
             '/3.0.0-beta.x/guides/draft',
+            '/3.0.0-beta.x/guides/schedule-article',
```

Comment:
We should rename to "Scheduled publication" to make it clear it's not for articles only

Reply:
Done @alexandrebodin